rust/ql/src/queries/security/CWE-770/UncontrolledAllocationSize.qhelp

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+<overview>
+
+<p>Allocating memory with a size based on user input may allow arbitrary amounts of memory to be
+allocated, leading to a crash or denial of service incident.</p>
+
+<p>If the user input is multiplied by a constant, such as the size of a type, the result may
+overflow. In a build with the <code>--release</code> flag Rust performs two's complement wrapping,
+with the result that less memory may be allocated than expected. This can lead to buffer overflow
```

Comment:
Tweaking the content so that less than is expressed first (for better readability).
```suggestion
with the result that less memory than expected may be allocated. This can lead to buffer overflow
```